.circleci/scripts/vs_install.ps1

```diff
@@ -13,15 +16,27 @@ $VS_INSTALL_ARGS = @("--nocache","--quiet","--wait", "--add Microsoft.VisualStud
 
 curl.exe --retry 3 -kL $VS_DOWNLOAD_LINK --output vs_installer.exe
 if ($LASTEXITCODE -ne 0) {
-    echo "Download of the VS 2017 installer failed"
+    echo "Download of the VS 2019 Version 16.7 installer failed"
     exit 1
 }
 
+$VS_UNINSTALL_ARGS = @("uninstall", "--installPath", "`"C:\Program Files (x86)\Microsoft Visual Studio\2019\BuildTools`"", "--quiet","--wait")
+
+if (Test-Path "C:\Program Files (x86)\Microsoft Visual Studio\2019\BuildTools") {
+    echo "start uninstalling"
+    Start-Process "${PWD}\vs_installer.exe"   -ArgumentList $VS_UNINSTALL_ARGS -NoNewWindow -Wait -PassThru
+}
+if (Test-Path "C:\Program Files (x86)\Microsoft Visual Studio\2019\BuildTools\VC") {
```

Comment:
Sorry, I don't get why do you need L29-L31 here. Do we need to uninstall MSVC twice?

Reply:
Removed. It was for testing